### PR TITLE
Fix bug on distributed training in mnist using MirroredStrategy API

### PR DESCRIPTION
**Pull request by parkjaeman**
I tried to run distributed tensorflow with mnist but it did not work. So I fixed such problem with MirroredStrategy API.

### CODE REVIEW

**Comment by parkjaeman** [comment]
I signed it!

**Comment by karmel** [comment]
Thanks for the PR, @parkjaeman . @robieta , @guptapriya -- can you take a look?

File: official/mnist/mnist.py [hunk 1]
```diff
@@ -125,7 +125,10 @@ def model_fn(features, labels, mode, params):
 
     logits = model(image, training=True)
     loss = tf.losses.sparse_softmax_cross_entropy(labels=labels, logits=logits)
-    accuracy = tf.metrics.accuracy(
+    if params.get('multi_gpu'):
+      accuracy = (tf.no_op(), tf.constant(0))
```
Comment (robieta) [inline official/mnist/mnist.py]: This should not be necessary. If you are having issues make sure your version of tf-nightly is up to date.

Comment (parkjaeman) [reply official/mnist/mnist.py]: I tried to change from mirrored strategy to distribution_utils.get_distribution_strategy. While mirroed strategy needs just num_gpus, distribution_utils.get_distribution_strategy might need all_reduce_algo, inter_op_parallelism_threads and intra_op_parallelism_threads. I am not sure how to set and pass such parameters. Please let me know about them.

**Pull request merged**